backend/src/main/java/com/zpi/backend/game/GameRepository.java

```diff
@@ -35,23 +39,23 @@ public interface GameRepository extends JpaRepository<Game, Long> {
     @Query(
             value = "select  G.id, original_id, G.name, age, min_players, max_players, playing_time," +
                     "short_description, image,game_status_id from games G join game_statuses GS on G.game_status_id = GS.id" +
-                    " where G.id = :id and GS.status = 'Accepted'",
+                    " where G.id = :id and GS.status = '"+ACCEPTED+"'",
```

Comment:
what are the pluses for?

Reply:
Because it is external constant and it is needed to concatenate like that